modules/swagger-codegen/src/main/resources/csharp/ApiClient.mustache

```diff
@@ -145,7 +145,7 @@ namespace {{packageName}}.Client
         /// <param name="contentType">Content Type of the request</param>
         /// <returns>Object</returns>
         public Object CallApi(
-            String path, RestSharp.Method method, Dictionary<String, String> queryParams, Object postBody,
+            String path, RestSharp.Method method, List<KeyValuePair<String, String>> queryParams, Object postBody,
```

Comment:
Thanks for the feedback. I agree that we should not change the contracts on the public methods if we don't need to. I think we can just  call a toDictionary() before calling the public ApiClient.CallApi() method. This also means that only one file needs to be modified, api.mustache. I can work on pushing out those changes today. I'll also make sure to update the CallAsync methods as well since I overlooked that in my initial commit (was using targetFramework v3.5).